packages/mirrorful/package.json

```diff
@@ -19,7 +19,8 @@
     "release:patch": "rm -rf ./dist; npm version patch; tsc --build; cd editor; yarn build; cd .next; rm -rf cache; cd ../..; npm publish;",
     "release:minor": "rm -rf ./dist; npm version minor; tsc --build; cd editor; yarn build; cd .next; rm -rf cache; cd ../..; npm publish;",
     "release:major": "rm -rf ./dist; npm version major; tsc --build; cd editor; yarn build; cd .next; rm -rf cache; cd ../..; npm publish;",
-    "testprod": "rm -rf ./dist; tsc --build; cd editor; yarn build; cd .next; rm -rf cache; cd ../..; npm link; echo '‼️ Now go to a test project and run npm link mirrorful; yarn run mirrorful'"
+    "testprod": "rm -rf ./dist; tsc --build; cd editor; yarn build; cd .next; rm -rf cache; cd ../..; npm link; echo '‼️ Now go to a test project and run npm link mirrorful; yarn run mirrorful'",
+    "prepare": "cd .. && cd .. && husky install packages/mirrorful/.husky"
```

Comment:
What does this script do / where does it get called?

Reply:
It generates the ```.husky``` dir with the required files inside ```packages/mirrorful/ ```

Reply:
Ah got it! So this is like a one time thing that needs to be run?

Also looks awesome, Arneesh — thanks for taking this on!

Reply:
I am confused why this command is added to the scripts section if you are pushing up `packages/mirrorful/.husky` ? It would appear that no one should need to ever run this?

Excellent work btw, thank you!